functions/DynamicParams.ps1

```diff
@@ -271,6 +271,55 @@ Function Get-ParamSqlLogins
 	return $newparams
 }
 
```

Comment:
Please remove from PR. I added i to DynamicParams with fixes; my bad, I didn't see it in this PR when I pulled your branch.